io.github.fasthud.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.fasthud",
```

Comment:
the app-id is incorrect. it doesn't include the user / oraganization.

Reply:
it needs to be changed, including upstream.

Reply:
done